cockatrice/src/dlg_forgotpasswordchallenge.cpp

```diff
@@ -0,0 +1,100 @@
+#include <QLabel>
+#include <QCheckBox>
+#include <QGridLayout>
+#include <QHBoxLayout>
+#include <QDialogButtonBox>
+#include <QMessageBox>
+#include <QDebug>
+
+#include "dlg_forgotpasswordchallenge.h"
+#include "settingscache.h"
+
+DlgForgotPasswordChallenge::DlgForgotPasswordChallenge(QWidget *parent)
+    : QDialog(parent)
+{
+
+	QString lastfphost; QString lastfpport; QString lastfpplayername;
+	lastfphost = settingsCache->servers().getHostname("cockatrice.woogerworks.com");
```

Comment:
BTW,  I havent looked at the code for the getHostname function but the behavior I am seeing in my windows client is the dialog's present the last host I connected to and if there is none listed it returns the server name "cockatrice.woogerworks.com".  Not sure if that is what the expected behavior is nor do I know if something like a mac client behaves differently.